apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-index.presenter.tsx

```diff
@@ -18,14 +19,20 @@ export const VocabularyListIndexPresenter = ({
         name: ({ name }) => renderMultilingualTextCell(name),
     };
 
+    const matchers: Matchers<IVocabularyListViewModel> = {
```

Comment:
One downside of moving to multilingual text is that it becomes tougher to populate the (filterable property) select with each language as a filterable property. I think this is probably the desirable behaviour for the long-run. Could you add a story for this? It's doable, but we can wait until later.